src/Rings/mpoly-ideals.jl

```diff
@@ -1172,6 +1172,24 @@ function minimal_primes(I::MPolyIdeal; algorithm::Symbol = :GTZ, cache::Bool=tru
   return V
 end
 
+@doc raw"""
+    function minimal_primes_zero_dim(I::MPolyIdeal{QQPolyRingElem})
+    
+Return a vector containing the minimal associated prime ideals of `I`.
+
+Calls a specialized variant of `minimal_primes` for zero dimensional ideals 
+with coefficient ring over the rationals.
```

Comment:
```suggestion
Calls a specialized modular variant of `minimal_primes` for zero dimensional ideals 
with coefficient ring over the rationals.
```

Reply:
What should one take away from this additional information? That the result may not be correct anymore? Consider me a user/developer, who stumbles upon this method and reads the docstring.

Reply:
Any better hint for the knowledgable user that there is a modular approach and Singular-level parallelization going on in the background is welcome -- but it needs to be of a kind that does not disturb the average user.

(I might not even have included this function into the list of exported commands, if it had been my choice.... It is very often significantly inferior to the usual method from primdec.lib concerning time consumption due to the overhead.)

Reply:
I can already not understand why I should write specialized here. This alone gives no information. Specialized modular is even more confusing. Two comments:
1) This Singular function comes with final tests (depending on the value). I will check asap what precisely these checks are doing and whether they guarantee correctness (correctness in some cases).
2) As we will more and more rely on modular GB based methods, we should develop a general philosophy of telling the truth to the user. In contrast to Magma, we should not cheat here.

With respect to this PR, let us decide on how to handle this as soon as I have checked what the tests do.

Reply:
Indeed the word specialized should be deleted, whereas I would keep modular.

I strongly second the need for a general philosophy as stated in 2. .

W.r.t. this PR: What we definitely need are the two bug fixes.

Reply:
Thanks for the feedback. I removed specialized and added modular. 

So far I did not export the function because I could not determine a clear use-case for it.